packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.classNames.ts

```diff
@@ -1,33 +1,8 @@
 import { getDividerClassNames } from '../Divider/VerticalDivider.classNames';
-import { getMenuItemStyles, getStyles as getContextualMenuStyles } from './ContextualMenu.cnstyles';
 import { ITheme, mergeStyleSets } from '../../Styling';
 import { IVerticalDividerClassNames } from '../Divider/VerticalDivider.types';
 import { memoizeFunction } from '../../Utilities';
 
-export interface IContextualMenuClassNames {
```

Comment:
I refactored these to use `IProcessedStyleSet<IContextualMenuStyles>` and `IProcessedStyleSet<IContextualMenuItemStyles>` which may be a breaking change given that all `IStyle` fields are required including `subComponentStyles`.